report/sarif_test.go

```diff
@@ -76,7 +76,7 @@ func TestWriteSarif(t *testing.T) {
 			}
 			want, err := os.ReadFile(test.expected)
 			require.NoError(t, err)
-			assert.Equal(t, want, got)
+			assert.Equal(t, string(want), string(got))
```

Comment:
It's quite hard to read a diff of bytes.

Reply:
Tbh I think this works well enough. 

Converting both to string here makes the diff normal JSON instead of a stream of bytes.